app/app.js

```diff
@@ -5,6 +5,7 @@ import * as game from './routes/game.js';
 import * as decks from './routes/decks.js';
 import * as actions from './routes/actions.js';
 import * as discourse from './routes/discourse.js';
+import * as hello from './routes/hello.js';
```

Comment:
One additional (non-essential) nit, is that this can be `import hello from` if the `hello.js` function is `export default async function get`. The default keyword makes that possible.

Reply:
good to know.